api/v1/certificatepolicy_types.go

```diff
@@ -69,73 +66,122 @@ func (t Target) String() string {
 	return fmt.Sprintf(fmtSelectorStr, t.Include, t.Exclude, *t.MatchLabels, *t.MatchExpressions)
 }
 
-// CertificatePolicySpec defines the desired state of CertificatePolicy
+// CertificatePolicySpec defines the certificates to be checked and the parameters within which
+// those certificates should fall.
```

Comment:
Sorry for missing this the first time. Fall where? Do you mean the certificate policies will be removed? 
```suggestion
// CertificatePolicySpec defines the certificates to be checked and the parameters within which
// those certificates should fall.
```

Reply:
As in the certificates targeted by the CertificatePolicy should be configured within the parameters set in the policy (otherwise they're noncompliant).

Reply:
Thanks Dale, but where do the policies go ? When I read "should fall", i interpreted 
```suggestion
// CertificatePolicySpec define which certificates need to be checked and the parameters within those policies
// are removed.
```

Reply:
The cert-policy-controller doesn't take any actions--it only reports compliance or noncompliance of certificates by comparing certificates with the spec in the CertificatePolicy, and it's up to the user to take actions to fix the certificates.

Reply:
@dhaiducek thanks Dale, I made an edit to my suggestion. let me know your thoughts

Reply:
Sure! Adding on to that, how about this:
```
// CertificatePolicySpec defines which certificates need to be checked and how those certificates
// should be configured to be compliant with the policy. Enforce remediation action is not
// currently supported, so you are responsible to take actions to fix any noncompliant certificates.
```